app/(auth)/layout.tsx

```diff
@@ -16,8 +14,10 @@ export const viewport: Viewport = {
 }
 
 export const metadata: Metadata = {
+  metadataBase: new URL(`https://${process.env.NEXT_PUBLIC_VERCEL_URL}` ?? 'http://localhost:3000'),
```

Comment:
Change this to compare witihn the string, as the `??` operator will always be truthey for the left-hand value here (it's in a string).

Reply:
Should be sorted now. Please can you take another look?